### ISSUE TITLE
Add a way to select new file (different IO)

### ISSUE THREAD

**Issue by Maijin** [radareorg/cutter#83]
![image](https://user-images.githubusercontent.com/1408600/32372228-4561ec4c-c094-11e7-8bbf-f9b7ad66b80b.png)

No need of special implementation on r2 side you can do `r2 http://google.com` already

probably a selector in this menu^ by default on "file on disk" and with the following IO which are the one that doesn't require Cutter refactoring:
* http
* ihex

And in future `oLj` result:
Require https://github.com/radare/radare2/issues/8773 another issue to be created

**Comment by thestr4ng3r** [comment]
Stuff like `ihex:///path/to/file` can now be entered and works.
But there is no real gui for yet, so it is not that obvious if you don't know about it.

**Comment by xarkes** [comment]
Yes, I think a dropdown menu should be enough (retrieve it through r2 -L)

**Comment by Maijin** [comment]
no you won't be able to use r2 -L because that means you supports all of them

**Comment by xarkes** [comment]
Don't we?

**Comment by Maijin** [comment]
No unless Cutter supports debugger

**Comment by xarkes** [comment]
Hmm k.

**Comment by Maijin** [comment]
Maybe the better is to use `oLj` and remove the one with `d` permission ?
